### DISCUSSION TITLE
How to you type C- M- commands?

### DISCUSSION THREAD

**Discussion by Esnos33** [meow-edit/meow#587] [Q&A] [answered]
Hi, is there an meow way to type C- M- commands in keypad mode? For example command for `comint-clear-buffer` is `C-c M-o`. I don't know if this is related, but I get which-key minibuffer when I type `C-c` by hand and I get meow's which-key minibuffer when I type `SPACE c`.

**Comment by eshrh** [comment]
Yes, it's `SPACE g`. See:

https://github.com/meow-edit/meow/blob/99e08c92bb5d8a695062ce53e2cffeffd3a058a6/meow-tutor.el#L599-L604

**Reply by Esnos33** [reply]
But I don't want `C-M-o`, I want `C-c M-o`. Comint shows me `split-line` for `C-M-o`.

**Reply by eshrh** [reply, marked as answer]
Sorry, I read your post too fast!

In this case doesn't just `SPACE c m o` work?

**Reply by Esnos33** [reply]
Wow, it works, didn't expect that, thanks for help
